src/server/pfs/server/server_test.go

```diff
@@ -761,8 +783,17 @@ func TestListCommit(t *testing.T) {
 	}
 }
 
+func silenceLogs() {
+	if !testing.Verbose() {
+		lion.SetLogger(lion.DiscardLogger)
+	}
+}
```

Comment:
Wait so does this mean if a test fails and we didn't pass `-v` then we won't see the logs from `lion`?